macro/src/main/scala/monocle/macros/internal/Macro.scala

```diff
@@ -44,8 +44,8 @@ private[macros] class MacroImpl(val c: blackbox.Context) {
         )
       ) if termDefName.decodedName.toString == termUseName.decodedName.toString =>
         c.Expr[Lens[A, B]](
-          typesFields.map{ case (t,f) => q"monocle.macros.GenLens[$t](_.$f)" }
-                     .reduce((a,b) => q"$a composeLens $b")
+          typesFields.map{ case (t,f) => q"_root_.monocle.macros.GenLens[$t](_.$f)" }
```

Comment:
Including `_root_` in fully-qualified paths is a macro hygiene thing. It ensures that the generated code can't clash with anything else called `monocle` that happens to be in scope at the call site.

Reply:
Nice, I didn't know why we had this in the code base.